src/css/vocabulary/vocabulary.css

```diff
@@ -127,12 +127,49 @@ header {
     cursor: pointer;
 }
 
-.masthead .primary-menu {
-    /* margin-top: 45px; */
-    /* vertical-align: bottom; */
-    
+/* style product identity typesetting */
+.masthead .identity-logo.product {
+    width: initial;
+    position: relative;
+    text-indent: 42px;
+    padding-top: 7px;
+    box-sizing: border-box;
+
```

Comment:
Yes! that's what the empty separator is doing at current ❤️ 

I very much do see your points, and I think they're valid.

But.... if you will allow some background context:

_(Apologies for the longer comment here, but it's a good chance to document some of this in a broader sense.)_

Ordering the lines is a much more complex sub-project I haven't tackled just yet because the order of the `property:value` pair declarations in a style rule is the order (generally) they are applied/executed, which means there are situations where certain properties need to be set before others for varied reasons: cross-browser quirks, flash of un-styled content bugs, performance, and even feature fallbacks. This makes CSS particularly powerful, but also (at times) annoyingly resistant to easy alphabetizing of properties on the whole or other simple line order procedures. There's also "folk" conventions of what things should be together and what order they should be in for "readability and modification based on what they do", like putting `height` and `width` together in the order because they are so heavily tied together, often changed together, calculated together at render, and so on. With that much variation I haven't devoted time to a sensible documented "guideline" for our CSS here just yet.

I want there to be reasonable guidelines for CSS file org, but haven't applied much yet because I want to get most everything in there, and also find some pre-existing standards/guides/etc. we could point to, or fallback on. Ordering `property:value` pairs in CSS is like alphabetizing lines in a programming function, a better read order may break expected outcomes of function, and I want a sound through-line to lean on for what we do, before I start trying to do something so I can more easily document it, and even lint it if possible. Which is why for the time being I've just split them into two groups only, but do plan to do more org in this area :)

For now, if it's Ok with you I'd like to: 

1. Add a comment at the top of the CSS file explaining the layout style/visual style section split, avoid adding a comment for every section just yet, since I worry it makes the properties harder to scan with its repeat occurrence (maybe unfounded), and it adds more lines to the file overall. Not against it, just want to do a bit more work in the styles first, and see if there's an existing sound convention to lean on in this space. 
2. Draft an Issue to explore some sensible commenting, grouping, and order we could lean on or build from, or even easily implement with a linter as its own pursuit a little further on; without sacrificing the "execution" order benefits we can get out of how we handle our `property:value` pairs. Handle that near the end of some of the page design implementations when we've got 80+% of our style rules in place and have solid bits to work from.

I am also, of course, open to other routes of action too if you have suggestions! 

I think one thing  creating the issue in item 2 might help with is we can collect any notions of linters and standards we'd like to adopt, or borrow from and drop them there to research and build from.

Reply:
Relevant Issue here:
* #20